sig/transaction/membership.md

```diff
@@ -20,6 +19,9 @@
 - [@cfzjywxk](https://github.com/cfzjywxk)
 - [@bobotu](https://github.com/bobotu)
 - [@sticnarf](https://github.com/sticnarf)
+- [@jackysp](https://github.com/jackysp)
+- [@longfangsong](https://github.com/longfangsong)
+- [@ekexium](https://github.com/ekexium)
```

Comment:
Sorry,  @longfangsong and @ekexium haven't met the conditions to become the [committer](https://github.com/pingcap/community/blob/master/architecture/README.md#committer)

Reply:
Addressed, PTAL @AndreMouche